File: TestiranjeProjekat/Models/Prijava.cs
namespace TestiranjeProjekat.Models
{
    public class Prijava
    {
        public int Id { get; set; }
        public string NazivTima {  get; set; }
        public int PotrebanBrojSlusalica {  get; set; }
        public int PotrebanBrojRacunara { get; set; }
        public int PotrebanBrojTastatura {  get; set; }
        public int PotrebanBrojMiseva { get; set; }
        public List<PrijavaIgracSpoj> Igraci { get; set; }
        public Turnir Turnir { get; set; }
    }
}

File: frontend/src/app/shared/models/prijava.ts
import { Igrac } from './igrac';
import { Turnir } from './turnir';

export interface Prijava {
  id: number;
  nazivTima: string;
  potrebanBrojSlusalica: number;
  potrebanBrojRacunara: number;
  potrebanBrojTastatura: number;
  potrebanBrojMiseva: number;
  igraci: Igrac[];
  turnir: Turnir | null;
}
